baseplate/events/queue.py

```diff
@@ -170,6 +170,11 @@ def serialize_v2_event(event):
     return Serializer.serialize(_V2_PROTOCOL_FACTORY, event)
 
 
+class EventLogger(object):
+    def log(self, **kwargs):
```

Comment:
The interface could have explicit parameters right? How those parameters get used is up to implementation but what baseplate's gonna pass in is gonna stay constant? That seems more friendly to future development.

Reply:
There are probably some explicit parameters but different events don't have all the same inputs so it needs the `**kwargs`.

Reply:
Sure, but this interface is just used for the experiments system to send events right? Maybe we should call the ABC `ExperimentEventLogger`?

Reply:
Sure right now it is, but I think it's a really useful interface that we could use in other places as well and it seems like a waste to restrict it to experiments.

Reply:
OK, but maybe we should worry about those cases only once they come up? Feels a bummer to overgeneralize without information.

Reply:
I'm with Andrew on this; the code incoming today supports a very general logger that can take in "arbitrary" parameters (quoted, because it still relies on *some* formatting) and construct and send v2 events.

Reply:
feels a bit magical to me, but ok